readme.md

```diff
@@ -0,0 +1,3 @@
+# Reporter Bot
+#### A most excellent bot!
```

Comment:
I think there's too many number sign.

Reply:
If we do a single `# the subheading ends up *really* big.  Check it out in the preview!

Reply:
```suggestion
## A most excellent bot!
```